controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -479,7 +465,7 @@ def test_create_volume_get_volume_exception(self, storage_agent):
         self.servicer.CreateVolume(self.request, self.context)
         self.assertEqual(self.context.code, grpc.StatusCode.INTERNAL)
         self.assertTrue("error" in self.context.details)
-        self.mediator.get_volume.assert_called_once_with(volume_name, pool_id='pool1')
+        self.mediator.get_volume.assert_called_once_with(volume_name, pool='pool1')

```

Comment:
Why don't we use the `pool` parameter here as in line #418?

Reply:
done